svm/tests/mock_bank.rs

```diff
@@ -200,6 +205,30 @@ pub fn create_executable_environment(
         .write()
         .unwrap()
         .insert(Clock::id(), account_data);
+
+    // default rent is fine
+    let rent = Rent::default();
+
+    let mut account_data = AccountSharedData::default();
+    account_data.set_data(bincode::serialize(&rent).unwrap());
+    mock_bank
+        .account_shared_data
+        .write()
+        .unwrap()
+        .insert(Rent::id(), account_data);
+
+    // advance nonce asserts recent blockhashes is non-empty but then just gets the blockhash from ic
```

Comment:
nit: I wasn't sure what "ic" was, can you use the full name for invoke context here? :)